tests/browser/test/basic_block_factory_test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Node.js script to run Automated tests in Chrome, via webdriver.
+ */
+
+const webdriverio = require('webdriverio');
+const chai = require('chai');
+
+let browser;
+suite('Testing Connecting Blocks', function (done) {
+  this.timeout(0);
+  // Setup Selenium for each of the test
```

Comment:
```suggestion
  // Setup Selenium for all of the tests.
```

I think this only runs once at the beginning of the suite, so I think this comment is clearer. But feel free to push back on this! (as well as any of my other comments :P )